crypto/x509/x509_lu.c

```diff
@@ -886,18 +890,19 @@ void X509_STORE_set_check_issued(X509_STORE *ctx,
     ctx->check_issued = check_issued;
 }
 
-X509_STORE_CTX_check_issued_fn X509_STORE_get_check_issued(const X509_STORE *ctx)
+X509_STORE_CTX_check_issued_fn X509_STORE_get_check_issued(const X509_STORE *cx)
```

Comment:
> why not `store`?

Simply because `cx` is short enough for avoiding the obligation to spilt the line due to the 80-chars width limit. Or how about using `s` here?

Reply:
The obligation is not hard obligation and reasonable variable/argument names are more important than this obligation.

Anyway `xs` would work for me and I'd use it also for other X509_STORE * arguments in other functions instead of ctx as X509_STORE * is not a context.

Reply:
Good point - changed all such occurrences, also in `x509_vfy.h.in` and in the respective docs.